### DISCUSSION TITLE
How to add an `install-context.reg` for Pycharm?

### DISCUSSION THREAD

**Discussion by hccz95** [ScoopInstaller/Scoop#5731] [Help and Questions]
I installed pycharm-professional from scoop/extras, but could not find the right-click menu of pycharm.
Maybe we need a `install-context.reg` script for pycharm-professional, but where to find the docs for writing a `.reg` script?

**Comment by HUMORCE** [comment]
There is no docs on how to writing scripts, people write scripts based on the behaviour of the official installer.
